src/storage/mod.rs

```diff
@@ -2031,6 +2055,95 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_txn_prewrite_insert() {
```

Comment:
I think you can add a test in `src/storage/txn/process.rs` which checks the statistics returned by `Executor::process_write()`.

Reply:
But all other case about transaction are tested in `src/storage/mod.rs`.